packages/rest/src/sequence.ts

```diff
@@ -34,6 +39,27 @@ export interface SequenceHandler {
   handle(context: RequestContext): Promise<void>;
 }
 
+/**
+ * Options for a sequence
+ */
+export type SequenceOptions = {
+  /**
+   * Order of actions by phase
+   */
+  orderOfActions?: string[];
+  /**
+   * Custom list of actions by order
+   */
+  actions?: RestAction[];
+};
+
+const DEFAULT_ORDER_OF_ACTIONS = [
+  'reject',
+  'send',
+  'route',
+  'parseParams',
+  'invoke',
```

Comment:
How can LB4 applications customize the order of the actions?

Let's say I am an app developer and want to call a new action in my sequence, e.g. `authenticate`. What are the instructions?